static/css/additional-styles.css

```diff
@@ -35,6 +35,7 @@ code, pre, kbd {
 code, kbd {
   padding: .1rem .3rem;
   border: 1px solid var(--c-border);
+  white-space: nowrap;
```

Comment:
```suggestion
} 

code::before {
  content: '\00200B';
```

Reply:
this doesn't appear to work, it only removes the space before the `<code>`

Reply:
Python Infrastructure Status - Uptime History
{"component":{"status":"operational","name":"pypi.org - General","created_at":"2022-11-11T22:24:00.989Z","updated_at":"2024-03-12T20:19:49.666Z","position":1,"description":null,"showcase":true,"start_date":"2022-11-11T00:00:00.000Z","id":"06hwjcnhlz03","page_id":"2p66nmmycsj3","group_id":"dhqvcb8lvrh1"},"component_select_list":[{"name":"PyPI","value":"dhqvcb8lvrh1"},{"name":"PyPI - pypi.org - General","value":"06hwjcnhlz03"}],"is_group":false,"months":[{"name":"September","year":2023,"start_offset":5,"uptime_percentage":1.0,"days":[{"color":"#2fcc66","date":"2023-09-01T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-02T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-03T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-04T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-05T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-06T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-07T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-08T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-09T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-10T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-11T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-12T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-13T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-14T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-15T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-16T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-17T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-18T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-19T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-20T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-21T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-22T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-23T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-24T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-25T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-26T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-27T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-28T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-29T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-09-30T00:00:00.000Z","events":[],"p":0,"m":0}]},{"name":"October","year":2023,"start_offset":0,"uptime_percentage":1.0,"days":[{"color":"#2fcc66","date":"2023-10-01T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-02T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-03T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-04T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-05T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-06T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-07T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-08T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-09T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-10T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-11T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-12T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-13T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-14T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-15T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-16T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-17T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-18T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-19T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-20T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-21T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-22T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-23T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-24T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-25T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-26T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-27T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-28T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-29T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-30T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-10-31T00:00:00.000Z","events":[],"p":0,"m":0}]},{"name":"November","year":2023,"start_offset":3,"uptime_percentage":1.0,"days":[{"color":"#2fcc66","date":"2023-11-01T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-02T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-03T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-04T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-05T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-06T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-07T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-08T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-09T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-10T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-11T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-12T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-13T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-14T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-15T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-16T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-17T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-18T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-19T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-20T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-21T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-22T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-23T00:00:00.000Z","events":[],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-24T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-25T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-26T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-27T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-28T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-29T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0},{"color":"#2fcc66","date":"2023-11-30T00:00:00.000Z","events":[{"name":"PyPI new user registration temporarily suspended","code":"0th66lc1l8by"}],"p":0,"m":0}]}],"group_component_data":null,"renderUptimePatternFF":false}